visualize_client:
  hdf5_open_service_name: hdf5_segmentation_service
  visualize_service_name: visualize_cloud_service

hdf5_open_server:
  hdf5_open_service_name: hdf5_open_service
  hdf5_real_phoxi_open_service_name: hdf5_real_phoxi_open_service
  hdf5_segmentation_service_name: hdf5_segmentation_service
  hdf5_file_path: //home/ericlab/tsuchida/2022_11/annotation/segmentation/multi_object3426/multi_object.hdf5